scripts/cli.ts

```diff
@@ -0,0 +1,148 @@
+#! /usr/bin/env node
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { resolve, join } = require('path')
+const {
+  appendFileSync,
+  copyFileSync,
+  existsSync,
+  mkdirSync,
+  readFileSync,
+  rmSync,
```

Comment:
Optional suggestion.

```suggestion
  existsSync,
  mkdirSync,
  readFileSync,
  rmSync,
  renameSync,
```

Reply:
Keeping it as it is. Not much difference.